src/main.js

```diff
@@ -31,7 +31,7 @@ vm.$mount('#app');
 
 // Set theme according to url in development. e.g. ?theme=01
 if (process.env.NODE_ENV !== 'production') {
-  const themeId = getUrlParameter('theme');
+  const themeId = getUrlParameter('mixinTheme');
```

Comment:
todo: I think we should not use the prefix for this one. Since this key is used in the url, it makes not sense to prefix it.